orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfo.groovy

```diff
@@ -166,6 +166,16 @@ class PackageInfo {
     return request
   }
 
+  @CompileDynamic
+  Map getArtifactSourceBuildInfo(Map trigger){
+    if (trigger.buildInfo?.artifacts) {
```

Comment:
Might want to check if trigger isn't null  (`trigger?.`) since you do that in the below `if` block